content/docs/rfcs/66/README.md

```diff
@@ -0,0 +1,32 @@
+---
+slug: 66
+title: 66/WAKU2-METADATA
+name: Waku Metadata Protocol
+status: raw
+editor: Alvaro Revuelta <alrevuelta@status.im>
+contributors:
+---
+
+# Metadata Protocol
+
+Waku specifies a req/resp protocol that provides information about the node's medatadata.
+
+
+## Protocol id
+
+`/vac/waku/metadata/1.0.0`
+
+## Request
+
+```proto
+message WakuMetadataRequest {
+}
+```
+
+## Response
+
+```proto
+message WakuMessageResponse {
+  uint64 network_id = 1;
```

Comment:
I would make this:

```suggestion
  optional uint64 network_id = 1;
```

Note that `optional` in the protobuf doesn't necessarily mean that this field won't be mandatory on protocol level under certain circumstances. E.g. if this is a general metadata req-resp protocol it may not be necessary to include all information in every exchange.

Reply:
I would say the networkId will be always present.